modules/peers.js

```diff
@@ -29,7 +29,7 @@ let self;
 const __private = {};
 let definitions;
 
-const peerDiscoveryFrequency = 30000;
+const peerDiscoveryFrequency = 10000;
```

Comment:
Is there any thought process behind changing this value? Are you changing it for testing purposes on devnet or proposing that change for mainnet. 

When node startup, discovery should trigger and is triggering immediately afterwards 30s is a reasonable delay for this activity.

Reply:
This was experimental.
Reason of experiment was
- It was 5s originally
- 30s for discovery is I think it was good, but we thought updating status of peers every 30s is not enough for our current logic

but now we found the reason of connect/disconnect problem, it might not be related.